portal-impl/src/com/liferay/portal/metadata/TikaRawMetadataProcessor.java

```diff
@@ -67,7 +86,7 @@ public void setParser(Parser parser) {
 	protected Metadata extractMetadata(
 		String extension, String mimeType, File file) {
 
-		Metadata metadata = super.extractMetadata(extension, mimeType, file);
+		Metadata metadata = new Metadata();
```

Comment:
@guilhermedcamacho The logic of this method is changed. Before it will do a support check, after your change, it did not even though you added the support method in the current class.

Reply:
Hi @tinatian,

In this case, we needed to change the logic because the `super.extractMetadata` method was implemented in the removed `XugglerRawMetadataProcessor` class.  So, after looking at the `XugglerRawMetadataProcessor` code,  we realized that we could initialize the metadata variable with `new Metadata()` or just leave it null to maintain the previous behavior.

Thx